accounts/abi/abi.go

```diff
@@ -255,7 +255,7 @@ func UnpackRevert(data []byte) (string, error) {
 		return "", errors.New("invalid data for unpacking")
 	}
 	if !bytes.Equal(data[:4], revertSelector) {
-		return "", errors.New("invalid data for unpacking")
+		return fmt.Sprintf("%#x", data), nil
```

Comment:
The method docs for `UnpackRevert` says

> UnpackRevert resolves the abi-encoded revert reason.

If the data is _not_ an abi-encoded revert reason, this method _should_ return an error. So the change you've done is not right. 

IMO you should fix this on the caller side. If `UnpackRevert` fails, the caller can just use the raw data instead.

Reply:
okay, I have moved it to the caller side